### PR TITLE
@pepopowitz => [Badge] Extract component from GridItem and add to FillwidthItem

### PR DESCRIPTION
**Pull request by mzikherman**
This wound up being pretty straightforward. Basically extracted badges as their own component, and then used them in both places.

However, the one possible blocker (and why its a WIP), is that in the Fillwidth rails, we wind up getting artworks that are smaller than ones we generally have in any column/grid based layout.

For these smaller artworks, multiple badges shows up a little not great:

Screen Shot 2019-04-15 at 5 41 27 PM

I was fiddling (unsuccessfully) with the layout, to try to stack the badges in a column in these 'small' cases, will keep at it! Up for review in the meantime.

Closes https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=57&projectKey=DISCO&modal=detail&selectedIssue=DISCO-897

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit d8a7ba96b881f9c4292687f9815c5306ddd272fa

https://deploy-preview-2299--artsy-reaction.netlify.com

File: src/Components/Artwork/Badge.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          {includeBidBadge && (
+            <Label>
+              <Sans size="0">Bid{saleTimingHint}</Sans>
+            </Label>
+          )}
+          {is_acquireable && (
+            <Label>
+              <a
+                href={href}
+                style={{ textDecoration: "none", cursor: "pointer" }}
```
Comment (pepopowitz) [inline src/Components/Artwork/Badge.tsx]: Do these styles need to be applied inline, or can we use styled components like the other elements are using?

Comment (mzikherman) [reply src/Components/Artwork/Badge.tsx]: Good point, this might even be better off as a `Link` anyway. Will clean it up.

**Comment by artsyit** [comment]
:rocket: PR was released in v16.1.1 :rocket:

**Pull request merged**